.travis.yml

```diff
@@ -15,5 +15,10 @@ before_install:
   - npm install -g gulp
   - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
 
-script:
-  - gulp test --browserstack
\ No newline at end of file
+script: |
+  echo ${TRAVIS_PULL_REQUEST}
```

Comment:
stray echo.

Reply:
Added for testing purpose. Removing it soon.